src/main/java/org/example/TestApplication.java

```diff
@@ -48,8 +48,14 @@ public void run(final TestConfiguration configuration,
                         new ProjectDao())));
 
         environment.jersey().register(
+                new SalesEmpController(
```

Comment:
should this be in this PR?

Reply:
It was there from a previous team project we believe